src/Cake.Common/Tools/XUnit/XUnitAliases.cs

```diff
@@ -8,7 +8,14 @@
 namespace Cake.Common.Tools.XUnit
 {
     /// <summary>
-    /// Contains functionality related to running xUnit.net tests.
+    /// <para>Contains functionality related to running <see href="https://github.com/xunit/xunit">xunit</see> tests.</para>
+    /// <para>
+    /// In order to use the commands for this alias, include the following in your build.cake file to download and
+    /// install from NuGet.org, or specify the ToolPath within the <see cref="XUnitSettings" /> class:
+    /// <code>
+    /// #tool "nuget:?package=xunit.runner.console"
```

Comment:
true, but make clear it's for testing V1 assemblies and cref `XUnit2Aliases` for clarity?